src/components/common/FileUploadPreview.vue

```diff
@@ -1,17 +1,12 @@
 <template>
-  <v-form
-    ref="form"
-    class="file-upload-preview"
-    lazy-validation
-  >
+  <div class="file-upload-preview">
     <v-file-input
       id="file-input"
       v-model="fileUpload"
       :label="label"
       filled
       dense
       accept=".pdf"
-      :rules="fileUploadRules"
```

Comment:
Rules work with form validation to do various checks and report if valid, but we'll do it manually below instead.